Composer/packages/client/src/utils/lgUtil.ts

```diff
@@ -1,10 +1,45 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+import { DialogInfo, LgFile, LgTemplate } from '@bfc/shared';
+import difference from 'lodash/difference';
+
+import lgWorker from '../recoilModel/parsers/lgWorker';
+
+import { getBaseName } from './fileUtil';
+
 /**
- * lgUtil.ts is a single place use lg-parser handle lg file operation.
- * it's designed have no state, input text file, output text file.
- *
+ * Auto create placeholder template used in dialog but did not exist in <dialog>.<*locale>.lg
  */
 
-export * from '@bfc/indexers/lib/utils/lgUtil';
+export const createMissingLgTemplatesForDialogs = async (
```

Comment:
This fn is a good candidate for unit test.

Reply:
updated